fishjam/media_events/shared.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package fishjam.media_events;
+
+enum Encoding {
```

Comment:
Is there any difference between `Encoding` and `Variant`? `Encoding` is used in the `VariantBitrates` message. Should we rename one of them?

```
message VariantBitrates {
    media_events.Encoding rid = 1;
    int32 bitrate = 2;
}
```